tests/testthat/test-variation-across-groups.R

```diff
@@ -0,0 +1,356 @@
+                                '135.00','124.00','152.25','157.00','150.00','135.50'),
+                         Max = c('  5.0','  6.0','  2.4','  9.0','  6.5','  8.0',
+                                 '145.0','150.0','173.0','160.0','168.0','152.0'),
+                         stringsAsFactors = FALSE)
+  
+  rownames(dfRes) <- c()
+  rownames(expected) <- c()
+  
+  testthat::expect_equal(dfRes,expected)
+})
+
+test_that("Measure col missing from df gives correct error (when one 
+          specified)", {
+            df <- data.frame(Gender = c('F','M','M','M','M','F','F','F'),
+                             LOS = c(3.2,NA,5,1.3,2.4,4,9,5))
+            
+            testthat::expect_error(variationAcrossGroups(df = df,
+                                                 categoricalCols = "Gender",
+                                                 measureColumn = "LO"), # <-- error
+                                   "The measure column or one of the categorical cols is")
+            })
+
+test_that("Measure col missing from df gives correct error (when two 
+          specified)", {
+            df <- data.frame(Gender = c('F','M','M','M','M','F','F','F'),
+                             LOS = c(3.2,NA,5,1.3,2.4,4,9,5))
+            
+            testthat::expect_error(variationAcrossGroups(df = df,
+                                                 categoricalCols = "Gender",
+                                                 measureColumn = c("LOS","BP")), # <-- error
+                                   "The measure column or one of the categorical cols is")
+            })
+
+test_that("Categorical col missing from df gives correct error (when one 
+          specified)", {
+            df <- data.frame(Gender = c('F','M','M','M','M','F','F','F'),
+                             LOS = c(3.2,NA,5,1.3,2.4,4,9,5))
+            
+            testthat::expect_error(variationAcrossGroups(df = df,
+                                                 categoricalCols = "Gendr", # <-- error
+                                                 measureColumn = "LOS"),
+                                   "The measure column or one of the categorical cols is")
+            })
+
+test_that("Categorical col missing from df gives correct error (when two 
+          specified)", {
+            df <- data.frame(Gender = c('F','M','M','M','M','F','F','F'),
+                             LOS = c(3.2,NA,5,1.3,2.4,4,9,5),
+                             Age = c(23,45,63,42,78,32,15,65))
+            
+            testthat::expect_error(variationAcrossGroups(df = df,
+                                                 categoricalCols = c("Gender"),
+                                                 measureColumn = c("LOS","Ag")),
+                                   "The measure column or one of the categorical cols is")
+            })
+
+test_that("Measure col as strings gives correct error", {
```

Comment:
I really like how you did all of these tests. Checked for correct functionality, checked for correct error triggering. Great job!